_posts/2018-01-17-upgrading-volt-to-circleci-two.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Upgrading Volt to CircleCI 2.0"
+date: 2018-01-17
+comments: true
+author: jonallured
+categories: [ci]
+---
+I was really excited about CircleCI 2.0, especially the [workflows
+features][workflows]. It seemed to me that with the work they had done here,
+really complicated builds would be able to be configured in a way that made more
+sense than on 1.0. This was something that was causing me grief on one of our
+projects so I upgraded to 2.0. It was pretty hard to get green and once we did,
+we decided to downgrade back to 1.0. Here's why.
+
+[workflows]: https://circleci.com/docs/2.0/workflows/
```

Comment:
this doesn't show your link (this is footnote syntax) switch to `[workflows](https://circleci.com/docs/2.0/workflows/)`

Reply:
I'm not sure what you mean - I booted up the site locally and all my links (including this one) work as expected. Why do I need to switch to the other syntax?

Reply:
It wouldn't surprise me if the Markdown engine didn't force footnote link notation to go at the end of the document. This works as-is, the convention Orta and I happen to follow puts all links grouped together at the bottom. I don't have a strong opinion on forcing that convention onto other writers.

Reply:
The GitHub preview didn't include the text:

![screen shot 2018-01-19 at 1 20 41 pm](https://user-images.githubusercontent.com/49038/35165247-98caf2ac-fd1b-11e7-9357-8c584390321b.png)

Reply:
I guess different renderers do different things, if our site shows it - that's fine 👍